docs/docsite/rst/user_guide/vault.rst

```diff
@@ -424,6 +429,58 @@ When using :ref:`ansible-vault` commands that encrypt content (:ref:`ansible-vau
 only one vault-id can be used.
 
 
+.. _vault_securing_editor:
+Securing your editor
+````````````````````
+
+As noted above, the vault system relies on your configured ``$EDITOR`` which can be a source of disclosures, most editors have conviniences to prevent loss of data, but this normaly relies
+on extra plain text files that can have a clear text copy of your secrets. To prevent this you should consult your editor's manual to ensure you configure it to avoid disclosing secure data.
+
+
+vim
+^^^
+
+These are some well known options you can set in command mode to avoid cases of disclosure, be aware that there are probably more settings you'll need to modify to ensure security, specially when using plugins.
+
+The swapfiles are a kind of autosave in case of crash/interruption, you can disable this via::
+
+    set noswapfile
+
+Prevent creating backup files::
+
+    set nobackup
+    set nowritebackup
+
+The viminfo file copies data from your session, sometimes even secrets, but you can disable this for the current session like this::
+
+    set viminfo=
+
+Disable copying to the system clipboard::
+
+    set clipboard=
+
+
+Most of these you can also set in ``.vimrc`` for all files or just specific paths/extensions, see the vim manual for specifics.
+
+
+emacs
+^^^^^
+
+These options can be used to avoid common known cases of disclosure, this list is by now way complete, specially when you consider plugins and extensions.
```

Comment:
```suggestion
Use these options to avoid common known cases of disclosure,  but be aware that this list is note complete, especially when you consider plugins and extensions. 
```